src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -465,6 +467,36 @@ RequestContext<object> requestContext
 
             await requestContext.SendResult(result.First().ToString());
         }
+        private async Task HandleGetCommandRequest(
```

Comment:
Add single blank line between method definitions.

Reply:
I don't get how I missed this comment. This has been rectified.

Reply:
Sorry... One more comment I forgot to mention...

In the v2 branch, async functions have a been renamed to have Async at the end. Can you add it to this one?

```
HandleGetCommandRequest
```